ocean_lib/config.py

```diff
@@ -136,6 +137,12 @@ def __init__(self, filename=None, options_dict=None, **kwargs):
         elif options_dict:
             self._logger.debug(f"Config: loading from dict {options_dict}")
             self.read_dict(options_dict)
+        else:
+            filename = os.getenv(ENV_CONFIG_FILE)
```

Comment:
can you please add a test for this? I believe there are some configuration tests which can be adapted quickly.

Reply:
what happens if the ENV_CONFIG_FILE is empty? Will it try to open an invalid filename or None? Can we check that the file exists before loading it?

Also, is it a file local to us, or could the end user interfere and potentially send a huge file/overload the system?

Reply:
> please add a test for this.

OK. I added many more, much better now. c781cfe..e698292

> what happens if ENV_CONFIG_FILE is empty

Good point. I added a test for this, and give a ValueError when it happens.